community/installation-guides/panel/ubuntu2004.md

```diff
@@ -96,15 +96,15 @@ The default php-fpm configuration is fine to use and can be started and then ena
 commands below.
 
 ```bash
-systemctl enable php7.4-fpm
-systemctl start php7.4-fpm
+systemctl enable php8.0-fpm
+systemctl start php8.0-fpm
 ```
 
 ### Nginx
 Please check our [tutorial](/tutorials/creating_ssl_certificates.md) on generating SSL certificates for more information.
 
 #### SSL Configuration
```

Comment:
This should be changed to Webserver Configuration as that's where you are linking to. You can also remove the Nginx SSL guide above because it's part of the webserver configuration page.

> Follow [this guide](/panel/1.0/webserver_configuration.html) to configure your web server.